### ISSUE TITLE
[nvmf_device_removal_pci_remove] malloc corruption (largebin double linked list corrupted) results in abort

### ISSUE THREAD

**Issue by mgerdts** [spdk/spdk#2953]
# CI Intermittent Failure

## Link to the failed CI build

https://ci.spdk.io/results/autotest-per-patch/builds/100492/archive/nvmf-phy-autotest/build.log

## Execution failed at

```
00:13:00.265  Running I/O for 90 seconds...
...
00:13:00.280  [2023-03-21 07:16:04.093270] nvme_qpair.c: 580:nvme_qpair_abort_queued_reqs: *ERROR*: aborting queued i/o
00:13:00.280  [2023-03-21 07:16:04.093295] nvme_qpair.c: 558:nvme_qpair_manual_complete_request: *NOTICE*: Command completed manually:
00:13:00.280  [2023-03-21 07:16:04.093314] nvme_qpair.c: 243:nvme_io_qpair_print_command: *NOTICE*: READ sqid:1 cid:0 nsid:1 lba:215712 len:8 PRP1 0x0 PRP2 0x0
00:13:00.280  [2023-03-21 07:16:04.093329] nvme_qpair.c: 474:spdk_nvme_print_completion: *NOTICE*: ABORTED - SQ DELETION (00/08) qid:1 cid:0 cdw0:0 sqhd:0000 p:0 m:0 dnr:0
00:13:00.280  [2023-03-21 07:16:04.093415] nvme_ctrlr.c:1624:nvme_ctrlr_disconnect: *NOTICE*: [nqn.2016-06.io.spdk:system_eth7] resetting controller
00:13:00.280   07:16:08	-- target/device_removal.sh@1 -- # rm -f /var/jenkins/workspace/nvmf-phy-autotest/spdk/test/nvmf/target/try.txt
00:13:00.280   07:16:08	-- target/device_removal.sh@1 -- # kill -9 4065384
00:13:00.280  test/nvmf/target/device_removal.sh: line 1: kill: (4065384) - No such process
00:13:00.280    07:16:08	-- common/autotest_common.sh@1066 -- # trap - ERR
00:13:00.280    07:16:08	-- common/autotest_common.sh@1066 -- # print_backtrace
00:13:00.280    07:16:08	-- common/autotest_common.sh@1094 -- # [[ ehxBET =~ e ]]
00:13:00.280    07:16:08	-- common/autotest_common.sh@1096 -- # args=('--transport=rdma' 'test/nvmf/target/device_removal.sh' 'nvmf_device_removal' '--transport=rdma')
00:13:00.280    07:16:08	-- common/autotest_common.sh@1096 -- # local args
00:13:00.280    07:16:08	-- common/autotest_common.sh@1098 -- # xtrace_disable
00:13:00.280    07:16:08	-- common/autotest_common.sh@20 -- # '[' no '!=' yes ']'
00:13:00.280    07:16:08	-- common/autotest_common.sh@21 -- # PREV_BASH_OPTS=ehxBET
00:13:00.280    07:16:08	-- common/autotest_common.sh@22 -- # [[ ehxBET == *\x* ]]
00:13:00.280    07:16:08	-- common/autotest_common.sh@23 -- # XTRACE_DISABLED=yes
00:13:00.280    07:16:08	-- common/autotest_common.sh@25 -- # set +x
00:13:00.280  ========== Backtrace start: ==========
00:13:00.280  
00:13:00.280  in /var/jenkins/workspace/nvmf-phy-autotest/spdk/test/common/autotest_common.sh:1066 -> run_test(["nvmf_device_removal"],["test/nvmf/target/device_removal.sh"],["--transport=rdma"])
00:13:00.280       ...
00:13:00.280     1061		timing_enter $test_name
00:13:00.280     1062		echo "************************************"
00:13:00.280     1063		echo "START TEST $test_name"
00:13:00.280     1064		echo "************************************"
00:13:00.280     1065		xtrace_restore
00:13:00.280     1066		time "$@"
00:13:00.281     1067		xtrace_disable
00:13:00.281     1068		echo "************************************"
00:13:00.281     1069		echo "END TEST $test_name"
00:13:00.281     1070		echo "************************************"
00:13:00.281     1071		timing_exit $test_name
00:13:00.281       ...
00:13:00.281  in /var/jenkins/workspace/nvmf-phy-autotest/spdk/test/nvmf/nvmf.sh:71 -> main(["--transport=rdma"])
00:13:00.281       ...
00:13:00.281     66  			TCP_INTERFACE_LIST=("${net_devs[@]}")
00:13:00.281     67  			if ((${#TCP_INTERFACE_LIST[@]} > 0)); then
00:13:00.281     68  				run_test "nvmf_perf_adq" $rootdir/test/nvmf/target/perf_adq.sh "${TEST_ARGS[@]}"
00:13:00.281     69  			fi
00:13:00.281     70  		else
00:13:00.281  => 71  			run_test "nvmf_device_removal" test/nvmf/target/device_removal.sh "${TEST_ARGS[@]}"
00:13:00.281     72  		fi
00:13:00.281     73  		run_test "nvmf_shutdown" $rootdir/test/nvmf/target/shutdown.sh "${TEST_ARGS[@]}"
00:13:00.281     74  		# TODO: disabled due to intermittent failures. Need to triage.
00:13:00.281     75  		# run_test "nvmf_srq_overwhelm" $rootdir/test/nvmf/target/srq_overwhelm.sh "${TEST_ARGS[@]}"
00:13:00.281     76  	fi
00:13:00.281       ...
00:13:00.281  
00:13:00.281  ========== Backtrace end ==========
```

A core file is generated.  Here's the backtrace from the thread that hit the SIGSEGV.  The only other thread was waiting on `epoll_wait()`.

```
00:13:10.251  Using host libthread_db library "/usr/lib64/libthread_db.so.1".
00:13:10.251  Core was generated by `/var/jenkins/workspace/nvmf-phy-autotest/spdk/build/examples/bdevperf -m 0x4 -z'.
00:13:10.251  Program terminated with signal SIGSEGV, Segmentation fault.
00:13:10.251  #0  0x00007fb4cb9b9184 in _int_malloc () from /usr/lib64/libc.so.6
00:13:10.251  [Current thread is 1 (Thread 0x7fb4cae72a00 (LWP 4065384))]
...
00:13:10.252  Thread 1 (Thread 0x7fb4cae72a00 (LWP 4065384)):
00:13:10.252  #0  0x00007fb4cb9b9184 in _int_malloc () from /usr/lib64/libc.so.6
00:13:10.252  No symbol table info available.
00:13:10.252  #1  0x00007fb4cb9babd6 in calloc () from /usr/lib64/libc.so.6
00:13:10.252  No symbol table info available.
00:13:10.252  #2  0x00007fb4ccacedd5 in poller_register (fn=0x7fb4cda444ee <bdev_nvme_poll_adminq>, arg=0xb49ce0, period_microseconds=0, name=0x7fb4cdaa23f1 "bdev_nvme_poll_adminq") at thread.c:1646
00:13:10.252          thread = 0x879a40
00:13:10.252          poller = 0x7fb4cda48a6c <bdev_nvme_reset_ctrlr>
00:13:10.252          __PRETTY_FUNCTION__ = "poller_register"
00:13:10.252          __func__ = "poller_register"
00:13:10.252  #3  0x00007fb4ccacf9fa in spdk_poller_register_named (fn=0x7fb4cda444ee <bdev_nvme_poll_adminq>, arg=0xb49ce0, period_microseconds=0, name=0x7fb4cdaa23f1 "bdev_nvme_poll_adminq") at thread.c:1725
00:13:10.252  No locals.
00:13:10.252  #4  0x00007fb4cda44475 in bdev_nvme_change_adminq_poll_period (nvme_ctrlr=0xb49ce0, new_period_us=0) at bdev_nvme.c:1553
00:13:10.252  No locals.
00:13:10.252  #5  0x00007fb4cda4701b in nvme_ctrlr_disconnect (nvme_ctrlr=0xb49ce0, cb_fn=0x7fb4cda488d3 <bdev_nvme_reconnect_ctrlr>) at bdev_nvme.c:1797
00:13:10.252          rc = 0
00:13:10.252          __PRETTY_FUNCTION__ = "nvme_ctrlr_disconnect"
00:13:10.252  #6  0x00007fb4cda48b5a in bdev_nvme_reset_ctrlr (i=0x883c80, status=0) at bdev_nvme.c:2047
00:13:10.252          nvme_ctrlr = 0xb49ce0
00:13:10.252          __PRETTY_FUNCTION__ = "bdev_nvme_reset_ctrlr"
00:13:10.252  #7  0x00007fb4ccad745d in _call_completion (ctx=0x883c80) at thread.c:2506
00:13:10.252          i = 0x883c80
00:13:10.252  #8  0x00007fb4ccac7212 in msg_queue_run_batch (thread=0x879a40, max_msgs=8) at thread.c:841
00:13:10.252          msg = 0x20000402f200
00:13:10.252          count = 1
00:13:10.252          i = 0
00:13:10.252          messages = {0x20000402f200, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0}
00:13:10.252          notify = 1
00:13:10.252          rc = 0
00:13:10.252          __func__ = "msg_queue_run_batch"
00:13:10.252          __PRETTY_FUNCTION__ = "msg_queue_run_batch"
00:13:10.252  #9  0x00007fb4ccaca7a5 in thread_poll (thread=0x879a40, max_msgs=0, now=2996939719705392) at thread.c:1063
00:13:10.252          msg_count = 697779
00:13:10.252          poller = 0xaa5b364d909fa
00:13:10.252          tmp = 0x879400
00:13:10.252          critical_msg = 0x0
00:13:10.252          rc = 0
00:13:10.252  #10 0x00007fb4ccacb63f in spdk_thread_poll (thread=0x879a40, max_msgs=0, now=2996939719705392) at thread.c:1156
00:13:10.252          orig_thread = 0x0
00:13:10.252          rc = 0
00:13:10.252  #11 0x00007fb4ccebe01a in _reactor_run (reactor=0x879400) at reactor.c:903
00:13:10.252          thread = 0x879a40
00:13:10.252          lw_thread = 0x879d80
00:13:10.252          tmp = 0x928e50
00:13:10.252          now = 2996939719705392
00:13:10.252          rc = 0
00:13:10.252  #12 0x00007fb4ccebe68e in reactor_run (arg=0x879400) at reactor.c:941
00:13:10.252          reactor = 0x879400
00:13:10.252          thread = 0x7fb4cc7c3905 <rte_get_next_lcore+43>
00:13:10.252          lw_thread = 0x7fccacc1bc
00:13:10.252          tmp = 0x7ffe2189f4a0
00:13:10.252          thread_name = "reactor_2\000\000\000\200\000\000\000\320\364\211!\376\177\000\000\200\227\374\314\002\000\000"
00:13:10.252          last_sched = 0
00:13:10.252          __func__ = "reactor_run"
00:13:10.252  #13 0x00007fb4ccebf14f in spdk_reactors_start () at reactor.c:1053
00:13:10.252          reactor = 0x879400
00:13:10.252          i = 4294967295
00:13:10.252          current_core = 2
00:13:10.252          rc = 0
00:13:10.252          __func__ = "spdk_reactors_start"
00:13:10.252          __PRETTY_FUNCTION__ = "spdk_reactors_start"
00:13:10.252  #14 0x00007fb4cceb394e in spdk_app_start (opts_user=0x7ffe2189f810, start_fn=0x41c21e <bdevperf_run>, arg1=0x0) at app.c:771
00:13:10.252          rc = 0
00:13:10.252          tty = 0x0
00:13:10.252          tmp_cpumask = {str = '\000' <repeats 256 times>, cpus = "\004", '\000' <repeats 126 times>}
00:13:10.252          g_env_was_setup = false
00:13:10.252          opts_local = {name = 0x443953 "bdevperf", json_config_file = 0x0, json_config_ignore_errors = false, reserved17 = "\000\000\000\000\000\000", rpc_addr = 0x7ffe218a032e "/var/tmp/bdevperf.sock", reactor_mask = 0x7ffe218a0324 "0x4", tpoint_group_mask = 0x0, shm_id = -1, reserved52 = "\000\000\000", shutdown_cb = 0x41c5e3 <spdk_bdevperf_shutdown_cb>, enable_coredump = true, reserved65 = "\000\000", mem_channel = -1, main_core = -1, mem_size = -1, no_pci = false, hugepage_single_segments = false, unlink_hugepage = false, reserved83 = "\000\000\000\000", hugedir = 0x0, print_level = SPDK_LOG_INFO, reserved100 = "\000\000\000", num_pci_addr = 0, pci_blocked = 0x0, pci_allowed = 0x0, iova_mode = 0x0, delay_subsystem_init = false, reserved137 = "\000\000\000\000\000\000", num_entries = 32768, env_context = 0x0, log = 0x0, base_virtaddr = 35184372088832, opts_size = 216, disable_signal_handlers = false, reserved185 = "\000\000\000\000\000\000", msg_mempool_size = 262143, rpc_allowlist = 0x0, vf_token = 0x0}
00:13:10.252          opts = 0x7ffe2189f560
00:13:10.252          i = 128
00:13:10.252          __func__ = "spdk_app_start"
00:13:10.252  #15 0x000000000041da95 in main (argc=14, argv=0x7ffe2189fa08) at bdevperf.c:2667
00:13:10.253          opts = {name = 0x443953 "bdevperf", json_config_file = 0x0, json_config_ignore_errors = false, reserved17 = "\000\000\000\000\000\000", rpc_addr = 0x7ffe218a032e "/var/tmp/bdevperf.sock", reactor_mask = 0x7ffe218a0324 "0x4", tpoint_group_mask = 0x0, shm_id = -1, reserved52 = "\000\000\000", shutdown_cb = 0x41c5e3 <spdk_bdevperf_shutdown_cb>, enable_coredump = true, reserved65 = "\000\000", mem_channel = -1, main_core = -1, mem_size = -1, no_pci = false, hugepage_single_segments = false, unlink_hugepage = false, reserved83 = "\000\000\000\000", hugedir = 0x0, print_level = SPDK_LOG_INFO, reserved100 = "\000\000\000", num_pci_addr = 0, pci_blocked = 0x0, pci_allowed = 0x0, iova_mode = 0x0, delay_subsystem_init = false, reserved137 = "\000\000\000\000\000\000", num_entries = 32768, env_context = 0x0, log = 0x0, base_virtaddr = 35184372088832, opts_size = 216, disable_signal_handlers = false, reserved185 = "\000\000\000\000\000\000", msg_mempool_size = 262143, rpc_allowlist = 0x0, vf_token = 0x0}
00:13:10.253          rc = 1
00:13:10.253  
00:13:10.253  --
```

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mgerdts. log: https://ci.spdk.io/public_build/autotest-per-patch_100492.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_100613.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mgerdts. log: https://ci.spdk.io/public_build/autotest-per-patch_100829.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_100877.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_100904.html

**Comment by ksztyber** [comment]
[Bug scrub] @jimharris will take a look. We can also wait until asan/ubsan are enabled in the functional tests on the CI and see how the call stack looks after that.

**Comment by jimharris** [comment]
> [Bug scrub] @jimharris will take a look. We can also wait until asan/ubsan are enabled in the functional tests on the CI and see how the call stack looks after that.

I wasn't able to glean anything from the logs on this - clearly the heap is corrupted somehow but there are no obvious clues yet how.

I think it's better to just wait until we get ASAN/UBSAN enabled on the functional tests - that is likely to more immediately point to where the problem is.

I'll keep this assigned to me for now.

**Comment by jimharris** [comment]
Issue #3010 is effectively a duplicate of this issue, but @AlekseyMarchuk already found a key piece of data that hadn't been uncovered on this one yet.  Note that the failing call stack is the *exact* same, so these are surely the same issue.

Alexey reported:

>Looks like memory corruption. I'll take a look in a week
>00:13:28.753  malloc(): largebin double linked list corrupted (nextsize)

https://ci.spdk.io/public_build/autotest-per-patch_103308.html
https://ci.spdk.io/results/autotest-per-patch/builds/103308/archive/nvmf-phy-autotest/build.log

So closing #3010, and will continue work on this through this entry.

**Comment by AlekseyMarchuk** [comment]
I looked at the log and code and wasn't able to find anything helpful. Memory corruption occurred in bdevperf process in bdev_nvme during handling of the controller reset. That is pretty complicated flow to debug it without reproduction)
@jimharris you wrote that ASAN/UBSAN will be enabled for this test soon. Do you know approximately when it will be enabled?

**Comment by mikeBashStuff** [comment]
I believe it will be available whenever this series gets merged: https://review.spdk.io/gerrit/c/spdk/spdk/+/17143 but it is currently blocked by several other issues.

**Comment by karlatec** [comment]
To be more specific: issues 2949, 2992, 2996, 3061. Fixing these might uncover some more issues of course.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_109197.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_109201.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @slawekptak. log: https://ci.spdk.io/public_build/autotest-per-patch_109590.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @slawekptak. log: https://ci.spdk.io/public_build/autotest-per-patch_109739.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_109813.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_109841.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_109821.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_109856.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_109861.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_109935.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_109968.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_109972.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_109787.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_110485.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_110494.html

**Comment by tomzawadzki** [comment]
[Bug scrub] This issue requires ASAN enabled to properly figure out the cause. Right now the nvmf-phy-autotest does not have it enabled. We should prioritize enabling it on this job via issue #.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_111578.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_111607.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_111625.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_111719.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_111699.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_111703.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_111956.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_111976.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_112037.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_112000.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @ksztyber. log: https://ci.spdk.io/public_build/autotest-per-patch_112013.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @apaszkie. log: https://ci.spdk.io/public_build/autotest-per-patch_112076.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_112161.html

**Comment by xjjx** [comment]
Another instance of this failure.. https://ci.spdk.io/public_build/autotest-nightly_2955.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @glimchb. log: https://ci.spdk.io/public_build/autotest-per-patch_111465.html

**Comment by tomzawadzki** [comment]
[Bug scrub] Asan on most if not all jobs should be enabled soon, which might increase the occurrence of this issue. Leaving open for now, to see if that changes.

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v22.11_1546.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_118734.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_261.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_269.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_274.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_275.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_276.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_280.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_284.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_285.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_286.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_301.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_304.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_307.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_314.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_326.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_327.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_331.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_333.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_334.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_336.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_338.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-v23.11_342.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @DamiaSan. log: https://ci.spdk.io/public_build/autotest-per-patch_121376.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_121953.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3455.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3462.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-spdk-master-vs-dpdk-main_3468.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_122604.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mikeBashStuff. log: https://ci.spdk.io/public_build/autotest-per-patch_122626.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_122783.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @mmkayPL. log: https://ci.spdk.io/public_build/autotest-per-patch_122801.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_122894.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @tomzawadzki. log: https://ci.spdk.io/public_build/autotest-per-patch_122902.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @sberbz. log: https://ci.spdk.io/public_build/autotest-per-patch_122933.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @spdkci / Known Issue Detector. log: https://ci.spdk.io/public_build/autotest-nightly_3708.html

**Comment by spdkci** [comment]
Another instance of this failure. Reported by @jimharris. log: https://ci.spdk.io/public_build/autotest-per-patch_123460.html

**Comment by tomzawadzki** [comment]
This issue was initially reported as an intermittent failure, but has not been observed in the new SPDK-CI system. If this issue reoccurs or if clear reproduction steps can be provided that qualify it as a 'bug', please feel free to reopen it.
